src/zope/app/i18n/tests/test_configure.py

```diff
@@ -0,0 +1,29 @@
+##############################################################################
+#
+# Copyright (c) 2017 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""Configuration tests.
+"""
+import unittest
+
+from zope.component.testing import PlacelessSetup
+from zope.configuration import xmlconfig
+
+import zope.app.i18n
+
+class TestConfiguration(PlacelessSetup, unittest.TestCase):
+
+	def test_configure(self):
+		xmlconfig.file('ftesting.zcml', zope.app.i18n.tests)
```

Comment:
Those are tabs! 

(And  the reason is because I initially mistakenly completed the filename as `test_configure.zcml`, which started me out editing in the wrong mode.)